### PR TITLE
Doc/example dogbone

### PR DESCRIPTION
**Pull request by chrpetre**
adding a new example on static structural analysis

### CODE REVIEW

File: examples/02-tbrom_examples/06-TBROM_FEA_static_structural_optimization.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+""".. _ref_example_TBROM_FEA_static_structural_optimization:
+
+Static structural analysis and optimization using 3D field ROM
+--------------------------------------------------------------
+
+This example shows how PyTwin can be used to perform different types of static structural analysis using 3D field ROM.
+A static structural model of a dog bone is created with Ansys Mechanical. A fixed support is applied on the right hand
+side, while a remote force is supplied on the left hand side resulting in a stress field and associated displacement.
+Non-linear structcural steel is used a representative material. A static ROM has been generated out of the original
```
Comment (lboucin) [inline examples/02-tbrom_examples/06-TBROM_FEA_static_structural_optimization.py:31]: typo: structural

Comment (lboucin) [reply examples/02-tbrom_examples/06-TBROM_FEA_static_structural_optimization.py:31]: Suggestion: A non-linear structural analysis with plastic material behavior (using steel) is performed.

**Review by lboucin — APPROVED** [review]
LGTM, Thanks! I've just made a minor comment in the description of the example.

File: examples/02-tbrom_examples/06-TBROM_FEA_static_structural_optimization.py (hunk 1 above)
Comment (chrpetre) [reply examples/02-tbrom_examples/06-TBROM_FEA_static_structural_optimization.py]: updated thanks

**Pull request merged**